chainercv/transforms/image/random_expand.py

```diff
@@ -2,44 +2,39 @@
 import random
 
 
-def random_expand(img, bbox, max_ratio=4, fill=0):
+def random_expand(img, max_ratio=4, fill=0, return_params=False):
     """Expand image randomly.
 
     This method expands the size of image randomly by padding pixels. The
-    aspect ratio of the image is kept. The bounding boxes are move according
-    to the expansion.
+    aspect ratio of the image is kept.
```

Comment:
I think borrowing descriptions from the original paper is more intuitive.
It was a little bit difficult to understand the intent of this operation with the current description for people who have never seen this before.


Randomly place an image on a canvas whose edge length is at most :obj:`max_ratio` times larger than the original image. The canvas is filled by a value :obj:`fill` except for the region where the original image is placed.

This data augmentation trick is commonly used to create "zoom out" effect [1].